### PR TITLE
Tests for premisEventXMLToObject

### PR DESCRIPTION
**Pull request by damonkelley**
Partially completes #27

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

**Comment by somexpert** [comment]
The whitespace for the first like 6 or 7 tests is inconsistent, even though they are all very similar.

**Comment by somexpert** [comment]
:thumbsup:

File: tests/test_presentation.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        assert 'Unable to parse' in str(e)
+        assert 'into datetime object' in str(e)
+
+    def test_datetime_that_has_decimal_seconds(self, event_xml):
+        """Test that a datetime string that contains a decimal second is properly
+        converted to a datetime object.
+        """
+        _, xml = event_xml
+
+        # Replace the datetime string with a valid datetimestring that
```
Comment (ldko) [inline tests/test_presentation.py]: Should "datetimestring be "datetime" string"? Other than that, I give the PR :+1: .

**Pull request merged**